client/my-sites/plan-features/item.jsx

```diff
@@ -1,39 +1,67 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component } from 'react';
 import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Gridicon from 'components/gridicon';
 import InfoPopover from 'components/info-popover';
+import { abtest } from 'lib/abtest';
 
-export default function PlanFeaturesItem( { description, children, className } ) {
-	const renderTipinfo = () => {
-		if ( ! description ) {
-			return null;
-		}
+export default class PlanFeaturesItem extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			showPopover: false,
+			mouseOverTimeout: null,
+			mouseOver: false
+		};
+		this.mouseOver = this.mouseOver.bind( this );
+		this.mouseOut = this.mouseOut.bind( this );
+	}
 
+	renderTipinfo( description ) {
 		return (
```

Comment:
Could we get the `description` from `this.props`? It's then easier to guess where `description` comes from.